### PR TITLE
ci-operator: use the vanity URL for our config API

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: cmd/ci-operator/main.go [hunk 1]
```diff
@@ -156,7 +156,7 @@ const (
 	// annotationCleanupDurationTTL is the annotation for requesting namespace cleanup after the namespace has been active
 	annotationCleanupDurationTTL = "ci.openshift.io/ttl.hard"
 	// configResolverAddress is the default configresolver address in app.ci
-	configResolverAddress = "http://ci-operator-configresolver-ci.apps.ci.l2s4.p1.openshiftapps.com"
+	configResolverAddress = "https://config.ci.openshift.org"
```
Comment (alvaroaleman) [inline cmd/ci-operator/main.go:159]: Shouldn't this be defined in pkg/api/domain.go?

**Comment by stevekuznetsov** [comment]
@alvaroaleman updated

**Comment by alvaroaleman** [comment]
/hold
integration test failures are genuine

**Pull request merged**